shuffle/cli/src/shared.rs

```diff
@@ -12,39 +11,32 @@ use serde_generate as serdegen;
 use serde_generate::SourceInstaller;
 use serde_reflection::Registry;
 use std::{
+    collections::BTreeMap,
     fs,
     fs::File,
     io::Write,
     path::{Path, PathBuf},
+    str::FromStr,
     time::Duration,
 };
 use transaction_builder_generator as buildgen;
 use transaction_builder_generator::SourceInstaller as BuildgenSourceInstaller;
+use url::Url;
 
 pub const MAIN_PKG_PATH: &str = "main";
 const NEW_KEY_FILE_CONTENT: &[u8] = include_bytes!("../new_account.key");
+pub const LOCALHOST_NETWORK_NAME: &str = "localhost";
+pub const LOCALHOST_NETWORK_BASE: &str = "http://127.0.0.1";
```

Comment:
nit: Why `pub`?

Reply:
They are both being used in normalized_network method of main.